packages/forma-36-react-components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,62 +1,107 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, select } from '@storybook/addon-knobs';
 
 import Tooltip from './Tooltip';
+import Paragraph from '../Typography/Paragraph';
 import TextLink from '../TextLink';
+import notes from './README.md';
 
-storiesOf('Components/Tooltip', module)
-  .addParameters({
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  parameters: {
     propTypes: Tooltip['__docgenInfo'],
-    component: Tooltip,
-  })
-  .add('default', () => (
-    <div>
-      <Tooltip
-        place={select(
```

Comment:
@gui-santos will this make it a breaking change?
What if I want the tooltip to always show on the right?

Reply:
good point! I will take a look at how to tell popper this